js/map.js

```diff
@@ -0,0 +1,44 @@
+// Width and height
+var chart_width     =   1000;
+var chart_height    =   600;
+
+// Projection
+var projection = d3.geoMercator() //changing to geoAlbers makes the map disappear
+    .scale(1500)
+    .center([-91.34397, 32.25196])
+    .translate([chart_width / 2, chart_height / 2]);
+var path = d3.geoPath()
+    .projection(projection);
+
+// Create SVG
+var svg = d3.select("body")
+    .append("svg")
+    .attr("width", chart_width)
+    .attr("height", chart_height);
+
+var map = svg.append( 'g' )
+    .attr( 'id', 'map' );
+
+// Add map features
+d3.json('../cache/state_map.geojson', function(us_data){
```

Comment:
`../cache` isn't going to be available on CHS

Reply:
.....WUT :woman_facepalming: been operating under that assumption with this code.

Do you think I can push them to a `target/data` directory?

Reply:
I think data that you want access to in target needs to be `publish` or otherwise exported.

Reply:
see #37